BrainPortal/app/controllers/tags_controller.rb

```diff
@@ -109,7 +109,7 @@ def tag_params #:nodoc:
     user_id      = new_tag_attr[:user_id]
     group_id     = new_tag_attr[:group_id]
     new_tag_attr[:user_id]  = current_user.id           if !current_user.available_users.where(:id => user_id).exists?
-    new_tag_attr[:group_id] = current_user.own_group.id if !current_user.available_groups.where(:id => group_id).exists?
+    new_tag_attr[:group_id] = current_user.own_group.id if !current_user.assignable_groups.where(:id => group_id).exists?
```

Comment:
As a SiteManager should I be able to assign a tag to a UserGroup ?

Reply:
We'll say yes. It's basically when a site manager creates a tag for anotehr user of the site.